src/mslice/plotting/plot_window/slice_plot.py

```diff
@@ -438,7 +438,7 @@ def get_slice_cache(self):
 
     def get_cached_workspace(self):
         cached_slice = self.get_slice_cache()
-        return getattr(cached_slice, IntensityCache.get_slice_type(self.intensity_type))
+        return getattr(cached_slice, self.intensity_type.name.lower())
```

Comment:
I based the `enum` names off the properties of `cut` and `slice` so I think we can just use the `name` property of `enum` here and simplify?

Reply:
Oh yeah... I didn't think of that. Yes, this would make it much cleaner!